src/foam/gui/AppStyles.js

```diff
@@ -0,0 +1,81 @@
+
+foam.CLASS({
+  package: 'foam.gui',
+  name: 'AppStyles',
+  extends: 'foam.u2.View',
+
+  documentation: 'Generic CSS that can be included into the top level controller of foam app. Implement to foam class to use.',
+
+  css: `
```

Comment:
Can the DetailView and TableView CSS just be moved to the Detail and TableViews? Can the rest just be moved to the standard Controller?

Reply:
There are not detailview css here. But in regards to the TableView css, it'll overwrite all table view styles in other projects that use the tableview.

Reply:
The following look like they're for DetailView:

 .foam-u2-DetailView {
+      background: #fafafa;
+      border: 1px solid grey;
+    }
+    .foam-u2-DetailView .foam-u2-DetailView {
+      width: auto;
+      margin: inherit;
+    }
+    .foam-u2-DetailView-title {
+      background: #ddd;
+      color: gray;
+      padding: 6px;
+    }

Reply:
woops